src/dmd/traits.d

```diff
@@ -2069,7 +2070,43 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
         auto tup = new TupleExp(e.loc, exps);
         return tup.expressionSemantic(sc);
     }
+    //https://issues.dlang.org/show_bug.cgi?id=22291
+    if(e.ident == Id.parameters)
+    {
+        //No args are valid
+        if(e.args)
+        {
+            char[] contents = cast(char[]) e.args.toString();
+            contents = contents[1..$];
+            contents[$-1] = '\0';
```

Comment:
why are we doing null termination here?

Reply:
This is probably to substitute the hanging right parenthesis?